hiq/src/hiq/utils.py

```diff
@@ -79,7 +79,8 @@ def execute_cmd(
         env=None,
         error_file=None,
         append_error=False,
-) -> Union[str, List[str]]:
+        runtime_output=False,
+) -> Union[str, List[str], int]:
     """
     If verbose is true, print out input.
     If check is true, and the process exits with a non-zero exit code, a CalledProcessError exception will be raised.
```

Comment:
We can also add a comment saying "If runtime_output is true, ..." for better understanding.